ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/flow_tracking_settings.schema.yml

```
@@ -0,0 +1,53 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  flow_tracking_settings:
+    documentation_options:
+      table: wan-preview
+    description: |-
+      Define the flow tracking parameters for this topology.
+    type: dict
+    keys:
+      flow_tracker_name:
+        type: str
+        default: WAN-FLOW-TRACKER
```

following discussions with @ClausHolbechArista lets change this to `FLOW-TRACKER` (also in the code)
```suggestion
        default: FLOW-TRACKER
```